eng/pipelines/runtime.yml

```diff
@@ -722,6 +722,20 @@ jobs:
       liveRuntimeBuildConfig: release
       liveLibrariesBuildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
 
+  #
+  # PGO Build
+  #
```

Comment:
Don't forget to add the coreclr PGO build as well here so we have an instrumented runtime.